README.md

```diff
@@ -2,7 +2,54 @@
 
 A Go library to support Raspberry Pi HATs (Hardware Attached on Top), starting with the [Sense HAT](https://www.raspberrypi.org/products/sense-hat/).
 
+
+### Go on Pi
+
+How to prepare a MicroSD card for Go development on a Raspberry Pi.
+
+On your Mac/PC:
+
+* Download [Raspian Jessie Lite](https://www.raspberrypi.org/downloads/raspbian/)
+* Make a MicroSD card with [Apple Pi Baker for Mac](
+http://www.tweaking4all.com/hardware/raspberry-pi/macosx-apple-pi-baker/) or [follow these instructions](https://www.raspberrypi.org/documentation/installation/installing-images/README.md).
+
+Power up the Pi with a Keyboard and TV/Monitor attached:
```

Comment:
I'm curious if SSH is enabled by default.

Can you use `ssh pi@raspberrypi.local` from your Mac after it boots? If so, this section could be reworked.